transport-native-unix-common/src/main/java/io/netty/channel/unix/PeerCredentials.java

```diff
@@ -16,25 +16,36 @@
 
 package io.netty.channel.unix;
 
+import io.netty.util.internal.UnstableApi;
+
+import static io.netty.util.internal.EmptyArrays.EMPTY_INTS;
+
 /**
  * User credentials discovered for the peer unix domain socket.
  *
  * The PID, UID and GID of the user connected on the other side of the unix domain socket
  * For details see:
  * <a href=http://man7.org/linux/man-pages/man7/socket.7.html>SO_PEERCRED</a>
  */
+@UnstableApi
 public final class PeerCredentials {
     private final int pid;
     private final int uid;
-    private final int gid;
+    private final int[] gids;
 
     // These values are set by JNI via Socket.peerCredentials()
-    PeerCredentials(int p, int u, int g) {
+    PeerCredentials(int p, int u, int[] gids) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Consider using varargs for methods or constructors which take an array the last parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=pmd%3AUseVarargs)